build/lib/skglm/__init__.py

```diff
@@ -0,0 +1,6 @@
+__version__ = '0.4dev'
```

Comment:
@floriankozikowski those files should not be committed, delete build/lib and and it to .gitignore